torch/optim/_functional.py

```diff
@@ -238,3 +238,42 @@ def rmsprop(params: List[Tensor],
             param.add_(buf, alpha=-lr)
         else:
             param.addcdiv_(grad, avg, value=-lr)
+
+
+def adamax(params: List[Tensor],
+           grads: List[Tensor],
+           exp_avgs: List[Tensor],
+           exp_infs: List[Tensor],
+           state_steps: List[int],
+           eps: float,
+           beta1: float,
```

Comment:
Could we make these keyword only arguments? That will reduce the chance to mix up the order of these in the function call.

Reply:
Yeah that make sense, let me create a follow up PR to change that.

Reply:
Here is the PR: https://github.com/pytorch/pytorch/pull/56185